201-eventhub-create-namespace-geo-recoveryconfiguration/azuredeploy.parameters.json

```diff
@@ -13,6 +13,12 @@
     },
 	"aliasName": {
 	  "value": "GEN-UNIQUE"	
-	}
+	},
+	"locationSecondaryNamepsace": {
```

Comment:
remove these and use the defaultValue in the template

Reply:
Resolved